packages/components/forms/src/controlled-input-field/ControlledInputField.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { EventHandler, ChangeEvent, ReactNode } from 'react';
+import { cx } from 'emotion';
+import { ValidationMessage } from '@contentful/f36-validation-message';
+
+import { FormLabel } from '@contentful/f36-components';
+import { ControlledInput } from '@contentful/f36-inputs';
+import { HelpText } from '@contentful/f36-helptext';
+import type { ControlledInputProps } from '@contentful/f36-inputs';
+import { styles } from './ControlledInputField.styles';
+import { Box, CommonProps } from '@contentful/f36-core';
+
+export interface ControlledInputFieldProps extends CommonProps {
```

Comment:
You should extend BoxProps here instead to get the margin and padding props.

Reply:
Thanks, will do so :)